app/services/idv/steps/ssn_step.rb

```diff
@@ -30,6 +35,11 @@ def invalid_state_response
         mark_step_incomplete(:document_capture)
         FormResponse.new(success: false)
       end
+
+      def threatmetrix_session_id
+        return nil if !IdentityConfig.store.proofing_device_profiling_collecting_enabled
```

Comment:
Same as above, do we need to check `IdentityConfig.store.proofing_device_profiling_collecting_enabled` twice (here and in the `call` function)?